### DISCUSSION TITLE
Predict variables and use them as input

### DISCUSSION THREAD

**Discussion by begovegar** [PaddlePaddle/PaddleSpatial#35] [BaiduKDDCup2022] [answered]
Hi, we would like to know if we can predict, for example, the Wind Speed for the next 48 hours every 10min, and use these values as an input for the target variable (Patv) in our predictive model. 
I mean, use different models to predict values such as Temp or Wdir and use them as input in the final model to predict Patv.
We tougth we could, but after reading your reply to this question "Limitations of model input #12" we are in doubt.
Thanks in advance.

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

If the wind speed is solely based on prediction, you can use it. We only limit the model's input, and we do not give any restrictions on how to build the model by yourselves. 

Thanks a lot.

Regards
